modules/cluster-logging-rn-5.3.13.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-13"]
+= Logging 5.3.13
+This release includes link:https://access.redhat.com/errata/RHSA-2022:6882[RHSA-2022:68828-OpenShift Logging Bug Fix Release 5.3.13].
```

Comment:
I'm not sure if this is a problem, but the format for the links is not consistent across the document:

- This release includes [RHSA-2022:7434-OpenShift Logging Bug Fix Release 5.5.4](https://access.redhat.com/errata/RHSA-2022:7434).
- This release includes [OpenShift Logging Bug Fix Release 5.5.3](https://access.redhat.com/errata/RHBA-2022:6858).
- This release includes [OpenShift Logging Bug Fix Release 5.5.2](https://access.redhat.com/errata/RHBA-2022:6559). 
- This release includes [OpenShift Logging Bug Fix Release 5.5.1](https://access.redhat.com/errata/RHSA-2022:6344).
- This release includes [RHSA-2022:7435-OpenShift Logging Bug Fix Release 5.4.8](https://access.redhat.com/errata/RHSA-2022:7435).
- This release includes [OpenShift Logging Bug Fix Release 5.4.6](https://access.redhat.com/errata/RHBA-2022:6558).
- This release includes [RHSA-2022:6183-OpenShift Logging Bug Fix Release 5.4.5](https://access.redhat.com/errata/RHSA-2022:6183).
- This release includes [RHBA-2022:5907-OpenShift Logging Bug Fix Release 5.4.4](https://access.redhat.com/errata/RHBA-2022:5907).
- This release includes [RHBA-2022:5556-OpenShift Logging Bug Fix Release 5.4.3](https://access.redhat.com/errata/RHBA-2022:5556).